### ISSUE TITLE
Projection of non-square images

### ISSUE THREAD

**Issue by Anas-Khayata** [skyflynil/stylegan2#29]
Many thanks @skyflynil for sharing such an amazing work. 
I would like to ask you if you have a working version of projector.py file for non-square images. 

projector.py throwing exception for reshape operation at line: 122 if the model is trained for non-square images

When I have commented the lines: 122-123
#v = tf.reshape(v, [1, 1, sz//2, 2, sz//2, 2]) # Downscale
#v = tf.reduce_mean(v, axis=[3, 5])
it worked but gave wrong results.

Any suggestion for how to set a correct numbers in:
v = tf.reshape(v, [1, 1, sz//2, 2, sz//2, 2]) # Downscale 
Thanks

**Comment by Anas-Khayata** [comment]
I fixed the issue in projector.py to accept non-square images:
the solution is very simple :)
just divide v[3], v[2] and set the value instead of 2 in reshape command
